styles/NavBar.styles.js

```diff
@@ -0,0 +1,180 @@
+import Link from 'next/link'
```

Comment:
use small letters for naming as used earlier for file names i.e. navbar.styles.js